requirements.txt

```diff
@@ -1,81 +1,52 @@
+-i https://pypi.org/simple
 anyjson==0.3.3
-
-coverage==4.0.3
-
-coveralls==1.1
-
-Django>=1.11.23
-
-django-debug-toolbar==1.8
-
-django-redis-cache==1.7.0
-
-django-reversion==2.0.6
-
-django-user-agents==0.3.0
-
+atomicwrites==1.2.1
+attrs==18.2.0
+certifi==2019.6.16
+chardet==3.0.4
+coverage==4.5.3
+coveralls==1.8.1
+django-debug-toolbar==2.0
+django-redis-cache==2.0.0
+django-reversion==3.0.4
+django-user-agents==0.4.0
+django==2.2.2
```

Comment:
I think that we should get rid of requirements.txt altogether -- or at least not think of it as authoritative. Dependencies are built using pipenv and the Pipfile.  We can generate requirements.txt using `pipenv lock -r > requirements.txt`.

Reply:
The generated pipenv -> requirements.txt seems like a good approach.